source/draw/RasterMonochrome.ooc

```diff
@@ -106,7 +115,7 @@ RasterMonochrome: class extends RasterPacked {
 						result += distance sqrt()
 					}
 				}
-			result /= (this size x squared + this size y squared as Float sqrt())
+			result /= (sizeX squared + sizeY squared as Float sqrt())
```

Comment:
I'm pretty sure this is a bug (should be more like `(x^2 + y^2) sqrt()`) but fixing it fails the RasterMonochrome `lossy ascii serialization` test case.